crates/uv-dev/src/generate_cli_reference.rs

```diff
@@ -267,6 +268,15 @@ fn generate_command<'a>(output: &mut String, command: &'a Command, parents: &mut
     parents.pop();
 }
 
+fn emit_env_option(opt: &clap::Arg, output: &mut String) {
+    if opt.is_hide_env_set() {
+        return;
+    }
+    if let Some(env) = opt.get_env() {
+        output.push_str(&format!("\n[env: {}=]", env.to_string_lossy()));
```

Comment:
<img width="836" alt="Screenshot 2024-09-03 at 8 49 18 AM" src="https://github.com/user-attachments/assets/09c29271-2adb-4beb-97db-8e14e9af9bb4">

It feels a little weird that it's not styled (e.g. with code blocks?) and the `[env: ...]` format makes a little less sense for the documentation. I'm not quite sure what I want, but are you willing to play with the style a bit? Otherwise I can get to that eventually.

Reply:
sure! what about:
<img width="613" alt="image" src="https://github.com/user-attachments/assets/4113d0e7-e291-4fbb-aaf8-cf1fc2186dc5">

Reply:
some random styles 🙈 
<img width="735" alt="image" src="https://github.com/user-attachments/assets/00a8b9a7-b684-4fbd-a7c0-57d529f9e9b9">

Reply:
Like this?
<img width="720" alt="image" src="https://github.com/user-attachments/assets/3ca3ef2e-db8f-483c-8257-d1bcb4621a25">

Reply:
Looks reasonable. We can always iterate on this. Thanks!